docs/src/what_is_aleph.md

```diff
@@ -0,0 +1,207 @@
+## 2. How Rush Does it? What is Aleph?
+
+Rush implements  [Aleph Consensus Protocol](https://arxiv.org/abs/1908.05156) -- we highly encourage you to read the original paper, as it offers the best explanation of the ideas and does not require much background to understand. In this documentation we try to offer a self-contained sketch of the protocol, although, because of its compactness it might be harder to understand. For those who have read the paper we would like to note that there are certain minor differences between the paper and the actual implementation that we highlight below in a dedicated section. However, these do not affect any of the fundamental ideas behind Aleph or its security guarantees and are mostly tricks and implementation details.
```

Comment:
`we highlight below in a dedicated section.` - change below to later or generate a link

Reply:
Fixed.